### ISSUE TITLE
Readme for initramfs role includes sample playbook with non-matching description

### ISSUE THREAD

**Issue by rjo-uk** [redhat-cop/infra.lvm_snapshots#74]
In the following file: https://github.com/redhat-cop/infra.lvm_snapshots/blob/main/roles/initramfs/README.md

The description reads

> The following yaml is an example of a playbook that runs the role against a group of hosts named rhel and increasing the size of its boot partition by 1G. The boot partition is automatically retrieved by the role by identifying the existing mounted partition to /boot and passing the information to the script using the kernel_opts.

However, the playbook does not mention the group of hosts `rhel` nor does it alter `/boot`.  I think the description needs updating as well as the playbook task name?

**Comment by swapdisk** [comment]
Yes, I agree. Let me update it by removing this example playbook from the README and more clearly explaining how the role is used by other roles and citing those as examples.
